src/plone/api/tests/test_content.py

```diff
@@ -1447,3 +1447,90 @@ def test_get_view_view_not_found(self):
 
         for should_be_there in should_be_theres:
             self.assertIn((should_be_there + "\n"), str(cm.exception))
+
+    def test_get_vocabulary(self):
+        """Test getting a vocabulary by name."""
+        from plone.api.exc import InvalidParameterError
+        from plone.api.exc import MissingParameterError
+
+        # The vocabulary name must be given as parameter
+        with self.assertRaises(MissingParameterError):
+            api.content.get_vocabulary()
+
+        # Test getting a commonly available vocabulary
+        vocab = api.content.get_vocabulary(name="plone.app.vocabularies.PortalTypes")
+        self.assertTrue(vocab)
+        self.assertTrue(hasattr(vocab, "__iter__"))  # Verify it's iterable
```

Comment:
I would replace these lines with a `self.assertIsInstance`

Reply:
would you suggest importing `collections.abc` just to replace current approach?
i can try going with this 
![image](https://github.com/user-attachments/assets/f8617b16-928a-4605-bb98-90c41225a514)
but i'm not sure if vocabularies with length 0 are supported.

Reply:
Nope, sorry. I was referring to something like:

```suggestion
self.assertIsInstance(vocabulary, SimpleVocabulary)
```